src/Core/Banshee.WebBrowser/Banshee.WebSource/WebView.cs

```diff
@@ -147,9 +172,7 @@ private void LoadHome ()
                 if (!IsReady) {
                     IsReady = true;
                     var handler = Ready;
-                    if (handler != null) {
-                        handler (this, EventArgs.Empty);
-                    }
+                    handler?.Invoke (this, EventArgs.Empty);
```

Comment:
I'd have probably written `Ready?.Invoke` and I'm wondering whether that's thread-safe now I see your version, stick with this for now :+1: